gdbinit.py

```diff
@@ -96,6 +108,8 @@ def update_deps(file_path):
     bin_path = os.path.join(venv_path, "bin")
     os.environ["PATH"] = bin_path + os.pathsep + os.environ.get("PATH", "")
 
+    update_deps(__file__)
```

Comment:
FYI, we can leave this in the old location, there's no need to move this here.

Reply:
Okay thanks for letting me know, I thought it was to be moved as per changes in https://github.com/nikoshell/pwndbg/commit/0cbdb3df09abf463d56797eeec3288362e35fd6e